package.json

```diff
@@ -7,7 +7,7 @@
     "stream-chat-docusaurus": "./init.sh"
   },
   "scripts": {
-    "compress": "rm docusaurus.tar.gz; tar -czf docusaurus.tar.gz ./docusaurus",
+    "compress": "rm docusaurus.tar.gz; tar --exclude='./docusaurus/node_modules' -czf docusaurus.tar.gz ./docusaurus",
```

Comment:
there's a lot more than just node_modules that needs to be excluded

Reply:
Cool, do you which folders should be excluded and/or which ones should be included? If we have a list for that we can exclude/include what we need

Reply:
should they be included or excluded?